src/components/OnThisPageNav.vue

```diff
@@ -20,8 +20,7 @@
           class="base-link"
           @click.native="handleFocusAndScroll(item.anchor)"
         >
-          <WordBreak v-if="item.i18n">{{ $t(item.title) }}</WordBreak>
-          <WordBreak v-else>{{ item.title }}</WordBreak>
+          <span>{{ item.i18n ? $t(item.title): item.title }}</span>
```

Comment:
Would there even be a need for the span at all if we were to make this change?

Reply:
No, you are right.